src/parser/parse_tree.h

```diff
@@ -3267,6 +3275,9 @@ typedef struct pt_dblink_info
   PT_NODE *pwd;
   PT_NODE *qstr;		/* query string */
   PT_NODE *cols;		/* column definition  */
+  PT_NODE *pushed_pred;		/* pushed predicate from main query */
```

Comment:
Add the following to the pt_apply_dblink_table() function
`PT_APPLY_WALK(parser, p->info.dblink_table.pushed_pred, arg);`

For variables other than dblink_table.cols, actions will be taken on other variables that were already there through CBRD-23977.

Reply:
It looks not necessary because the pushed_pred is walk done.